ftw/simplelayout/contenttypes/browser/templates/textblock.pt

```diff
@@ -16,7 +15,6 @@
     <div tal:attributes="class image/wrapper_css_classes">
         <a tal:omit-tag="not: image/link_url"
            tal:attributes="href image/link_url;
-                           title image/link_title;
```

Comment:
Can you remove that from the python code too?
https://github.com/4teamwork/ftw.simplelayout/blob/9cbb2a3d3c211dd11f283bdab5f312584b09fc0f/ftw/simplelayout/contenttypes/browser/textblock.py#L44